interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -144,39 +144,39 @@ MathFuncts, errGen) {
         return this.getSampleData(val) || {id: val};
       }, this);
     },
-    getNodeObjects: function() {
+    getSignatureObjects: function() {
       // The heatmapData.activity array organizes activity data in a
       // representation convenient to render using vega.js: each element of the
       // array corresponds to one mark on the heatmap. For clustering by
       // hcluster.js, on the other hand, we need to reorganize the data so that
-      // all activity for each *node* is collected in an array. The result is
-      // essentially the same as that from `getSampleObjects` above, but
+      // all activity for each *signature* is collected in an array. The result
+      // is essentially the same as that from `getSampleObjects` above, but
       // transposed. We achieve this without too many intermediate steps via
       // two nested Array.prototype.map() operations:
 
-      // (1) first, we obtain a list of nodes by retrieving node activity
-      //     for the first sample in our heatmap
-      var firstSampleNodes = this.activityCache.get(
+      // (1) first, we obtain a list of signatures by retrieving signature
+      //     activity for the first sample in our heatmap
+      var firstSampleSignatures = this.activityCache.get(
         this.heatmapData.samples[0]
       );
-      // (2a) next, we build a new array (`retval`) comprised of `nodeObject`s
-      //      by walking through the `firstSampleNodes` and constructing a
-      //      `nodeObject` for each. [outer .map()]
-      var retval = firstSampleNodes.map(function(val) {
-        var nodeObject = {
+      // (2a) next, we build a new array (`retval`) comprised of
+      //      `signatureObject`s by walking through the `firstSampleSignatures`
+      //       and constructing a signatureObject` for each. [outer .map()]
+      var retval = firstSampleSignatures.map(function(val) {
+        var signatureObject = {
           'id': val.node,
           'activity': this.heatmapData.samples.map(
-            // (2b) the array of activity for each node is built by plucking the
-            //      activity `.value` for each sample within this node from the
-            //      `activityCache` [inner .map()]
+            // (2b) the array of activity for each signature is built by
+            //      plucking the activity `.value` for each sample within this
+            //      signature from the `activityCache` [inner .map()]
             function(sampleId) {
-              // FIXME: counting on array order to match node order here
+              // FIXME: counting on array order to match signature order here
               return this.activityCache.get(sampleId)[val.node - 1].value;
```

Comment:
This problem is actually separate from the renaming you did. If you want to proceed with this PR, let's file a separate issue. I will probably need to change several lines that are scattered around so it would be easier to fix that way anyway.